libs/SalesforceHybridSDK/SalesforceHybridSDKTests/ForceJSTestSuite.m

```diff
@@ -24,7 +24,11 @@
 
 #import <UIKit/UIKit.h>
 
-#import "ForceJSTestSuite.h"
+#import "SFPluginTestSuite.h"
```

Comment:
Unrelated to the react work - got rid of a bunch of almost empty header files for test classes by doing the interface declaration in the `.m`